templates/spambug.html

```diff
@@ -0,0 +1,24 @@
+<p>The following {{ plural('bug', data, pword='bugs') }} {{ plural('doesn\'t', data, pword='don\'t') }} are probably spam :
```

Comment:
nano-nit: `are` should be `is` when data contains only one bug

Reply:
And `{{ plural('doesn\'t', data, pword='don\'t') }}` should be removed I guess